src/test/java/com/rovio/ingest/DruidSourceBaseTest.java

```diff
@@ -84,7 +84,10 @@
     public File testFolder;
 
     @Container
-    public static MySQLContainer MYSQL = new MySQLContainer()
+    public static MySQLContainer MYSQL =
+            // MySQL 8 requires mysql-connector-java 8.x - test that
```

Comment:
Based on the PR description, this was already tested.

Reply:
This is not a TODO comment, if that's how you read it? By this comment I mean that we use "mysql:8.0.28" in tests from now on, because if we'd use the default MySQLContainer then we wouldn't cover the MySQL 8 case in testing.

Reply:
Would this be clearer:
```suggestion
            // MySQL 8 requires mysql-connector-java 8.x so we test against that
```
?

Reply:
Right, I misread it so that the sentence continued over to the next line. Much clearer now. To be very explicit, could even put a period at the end of the first sentence but it is clear enough now.